src/main/java/it/pagopa/interop/probing/eservice/operations/service/EserviceServiceImpl.java

```diff
@@ -45,6 +48,33 @@ public class EserviceServiceImpl implements EserviceService {
 	@Autowired
 	Validator validator;
 
+	@Override
+	public Long saveEservice(EserviceSaveRequest inputData) {
+		UUID eserviceId = UUID.fromString(inputData.getEserviceId());
+		UUID versionId = UUID.fromString(inputData.getVersionId());
+		List<String> basePath = inputData.getBasePath();
+		Eservice eServiceToUpdate = eserviceRepository.findByEserviceIdAndVersionId(eserviceId, versionId).orElse(null);
+
+		if (Objects.isNull(eServiceToUpdate)) {
+			eServiceToUpdate = new Eservice();
```

Comment:
Rid of new, use builder

Reply:
Done, I implemented the builder design pattern also on the dtos, in order to have a common standard. I just pushed, let me know if it is okay